src/arb/ComplexMat.jl

```diff
@@ -23,12 +23,6 @@ zero(m::ComplexMat, R::ComplexField, r::Int, c::Int) = similar(m, R, r, c)
 #
 ###############################################################################
 
-parent_type(::Type{ComplexMat}) = ComplexMatSpace
-
-elem_type(::Type{ComplexMatSpace}) = ComplexMat
-
-parent(x::ComplexMat) = matrix_space(base_ring(x), nrows(x), ncols(x))
-
 dense_matrix_type(::Type{ComplexFieldElem}) = ComplexMat
 
 base_ring(a::ComplexMatSpace) = ComplexField()
```

Comment:
All of these `base_ring` methods can go as they are now backed up by AA. This wasn't the case before this change as e.g. `ComplexMatSpace` does not contain the field as it is a singleton.

Reply:
Aha, I'll remove those shortly